franka.sh

```diff
@@ -2,7 +2,7 @@
 
 pushd /src/drake-franka-driver
 
-if [[ ! -v SIM_ROBOT ]]; then # SIM_ROBOT environment variable not set
+if [[ ${SIM_ROBOT:-false} == 'false' ]]; then
```

Comment:
should we bother with a default value? it should always be set in the config, right? might want to log/complain if it's not set at all